afids_utils/afids.py

```diff
@@ -0,0 +1,59 @@
+"""Anatomical fiducial classes"""
+from __future__ import annotations
+
+import attr
+import numpy as np
+import polars as pl
+from numpy.typing import NDArray
+
+from afids_utils.exceptions import InvalidFiducialError
+
+
+@attr.define
+class AfidSet(dict):
+    """Base class for a set of fiducials"""
+
+    slicer_version: str = attr.field()
+    coord_system: str = attr.field()
+    afids_df: pl.DataFrame = attr.field()
```

Comment:
I think my preferred approach would be to define an `Afid` class with a label and x, y, z coords, then make this a `list[Afid]`, and add a validator that it contains 32 `Afid`s with labels in order.

Reply:
Yes, we can certainly do that - it'd be similar to how we had it implemented in the validator. The validation logic is implemented, just not as a separate method (under `load`) currently. If we do switch to doing it that way, than I would say lets just get rid of the use of dataframes all together - I don't see foresee any use case for them and would save us an additional dependency.

Reply:
Sounds good on all counts, with the note that I think it makes sense to implement the validation logic as an attrs [validator](https://www.attrs.org/en/stable/init.html#validators) to check the validation regardless of where the AFIDs are coming from

Reply:
That works for me, the the validation logic currently in place, lets move the attrs validator as a thing to do during the sprint. Should be easy enough to take what is currently there.

Reply:
Cool, this has been done now with the latest commit _sans attrs validator_.